cps/db.py

```diff
@@ -49,6 +50,25 @@ def title_sort(title):
     Column('lang_code', Integer, ForeignKey('languages.id'), primary_key=True)
     )
 
+cc = conn.execute("SELECT id, datatype FROM custom_columns")
+cc_ids = []
+cc_exceptions = ['bool', 'datetime', 'int', 'comments', 'float', ]
```

Comment:
rating and enumeration works for me. i tested all types. could you post your metadata.db??